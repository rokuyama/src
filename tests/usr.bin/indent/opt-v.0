/* $NetBSD: opt-v.0,v 1.3 2021/10/08 20:23:23 rillig Exp $ */
/* $FreeBSD$ */

/*
 * A long comment.
 */
void
example(void)
{
	printf("A very long message template with %d arguments: %s, %s, %s", 3, "first", "second", "third");
}

/* $ This comment is neither counted nor formatted. */
#define macro1 /* prefix */ suffix

/* $ This comment is formatted and counted. */
#define macro2 prefix /* suffix */
