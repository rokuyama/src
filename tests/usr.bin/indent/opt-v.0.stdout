/* $NetBSD: opt-v.0.stdout,v 1.3 2021/10/08 20:23:23 rillig Exp $ */
/* $FreeBSD$ */

/*
 * A long comment.
 */
void
example(void)
{
	printf("A very long message template with %d arguments: %s, %s, %s", 3, "first", "second", "third");
}

#define macro1 /* prefix */ suffix

#define macro2 prefix		/* suffix */
There were 10 output lines and 2 comments
(Lines with comments)/(Lines with code):  0.571
